packages/contracts/contracts/libraries/LibVoting.sol

```diff
@@ -263,7 +263,7 @@ library LibVoting {
         bool isVoteInConsensus;
 
         for (uint256 i; i < numberOfMatchResults; i++) {
-            isVoteInConsensus = LibProofManager.verifyProof(dataHash, matchResults[i], dataProof);
+            isVoteInConsensus = LibProofManager.verifyProof(matchResults[i], dataHash, dataProof);
```

Comment:
so the `dataHash` is a leaf and the current `matchResult` the rootHash of the tree ?

Reply:
yes